app/helpers/container_image_helper/textual_summary.rb

```diff
@@ -0,0 +1,68 @@
+module ContainerImageHelper
+  module TextualSummary
+    #
+    # Groups
+    #
+
+    def textual_group_properties
+      items = %w(name tag id)
+      items.collect { |m| send("textual_#{m}") }.flatten.compact
+    end
+
+    def textual_group_relationships
+      items = %w(containers registry ems)
+      items.collect { |m| send("textual_#{m}") }.flatten.compact
+    end
+
+    #
+    # Items
+    #
+
+    def textual_name
+      {:label => "Name", :value => @record.name}
+    end
+
+    def textual_tag
+      {:label => "Tag", :value => @record.tag}
+    end
+
+    def textual_id
+      {:label => "Image Id", :value => @record.image_ref}
+    end
+
+    def textual_containers
```

Comment:
All these (`textual_containers`, `textual_registry`) can be removed if #3402 will be merged soon.